manual/src/cmds/profile-perf.etex

```diff
@@ -0,0 +1,90 @@
+\chapter{Profiling (perf)} \label{c:profiler-perf}
+%HEVEA\cutname{profiler-perf.html}
+
+This chapter describes how to use \texttt{perf}, the Linux Performance Events tool, to profile OCaml programs.
+
+Linux Performance Events (\texttt{perf (1)}) is a set of tools for performance observability, including CPU performance counter profiling and static and dynamic tracing. This chapter covers key features such as \texttt{perf-record(1)} for recording events and \texttt{perf-report(1)} for printing and visualising recorded events. \textt{perf} has many other features for profiling and visualising performance, see (man perf(1), \href{https://perfwiki.github.io/main/}{Perf wiki}, or \href{https://www.brendangregg.com/perf.html}{Brendan Gregg's Blog}) for more general documentation.
```

Comment:
`man` is the unix help tool for the command line. Try typing `man man` in a terminal on your Mac, (`q` to quit).

This help is traditionally broken down in numbered sections. The sections of the manual are:
1.   General Commands Manual
2.   System Calls Manual
3.   Library Functions Manual
4.   Kernel Interfaces Manual
5.   File Formats Manual
6.   Games Manual
7.   Miscellaneous Information Manual
8.   System Manager's Manual
9.   Kernel Developer's Manual

This is often summarised in documentation as `perf (1)`. So you would type `man perf` to see the documentation. This is assumed knowledge of how man pages work on Unix systems, I think it is fine to continue with this convention and rely on people doing a search if they don't understand. 

For this document I'll introduce the command reference as `perf (1)` etc and then use `perf` later on when referring to the command or Perf when talking about the general tool to avoid repeating "Linux Performance Events"